Assets/Mirror/Runtime/Transport/FallbackTransport.cs

```diff
@@ -0,0 +1,137 @@
+// uses the first available transport for server and client.
+// example: to use Apathy if on Windows/Mac/Linux and fall back to Telepathy
+//          otherwise.
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Mirror
+{
+    public class FallbackTransport : Transport
+    {
+        public Transport[] transports;
+
+        // the first transport that is available on this platform
+        Transport available;
+
+        public void Awake()
+        {
+            if (transports == null || transports.Length == 0)
+            {
+                Debug.LogError("FallbackTransport requires at least 1 underlying transport");
+            }
+            InitClient();
+            InitServer();
+            available = GetAvailableTransport();
+            Debug.Log("FallbackTransport available: " + available.GetType());
```

Comment:
reduce the noise on the console,  most transports already report when they are up, so this is redundant

Reply:
true, 'most' do.
but if one doesn't, then it's not obvious which one the Fallback transport used.
would make debugging a whole lot more difficult for people.